axelor-supplychain/src/main/java/com/axelor/apps/supplychain/module/SupplychainModule.java

```diff
@@ -165,5 +168,7 @@ protected void configure() {
     bind(TimetableService.class).to(TimetableServiceImpl.class);
     bind(InvoiceServiceSupplychain.class).to(InvoiceServiceSupplychainImpl.class);
     bind(StockMoveServiceSupplychain.class).to(StockMoveServiceSupplychainImpl.class);
+    bind(StockLocationServiceImpl.class).to(StockLocationServiceSupplychainImpl.class);
+    bind(StockLocationServiceSupplychain.class).to(StockLocationServiceSupplychainImpl.class);
   }
```

Comment:
add bind for interface to interface too.

Reply:
sorry, binding of interface will not work(as it was already bind), so just don't add it.